webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/nodes/NodeInfo.scala

```diff
@@ -93,8 +93,8 @@ final case class NodeInfo(
   val isSystem                   = node.isSystem
   val isPolicyServer             = node.isPolicyServer
   val creationDate               = node.creationDate
-  val nodeReportingConfiguration = node.nodeReportingConfiguration
-  val properties                 = node.properties
+  def nodeReportingConfiguration = node.nodeReportingConfiguration
```

Comment:
Can you add a comment, else it will be lost the nexttime we touch that code

Reply:
it's not definitive, i'm only testing stuff